linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/bash/shared.sh

```diff
@@ -7,6 +7,11 @@ disable_prelink
 
 package_install dracut-fips
 
+# Enable AESNI if supported
+if grep -q -m1 -o aes /proc/cpuinfo; then
```

Comment:
Preferentially, this would be in an OVAL file and added to the OVAL grub FIPS check using the OVAL and bash templates.

Reply:
This is a preferred configuration, but not a security requirement. I would think that the OVAL should only check that the security requirement is met.

Reply:
OVAL is more than about meeting security requirements. Please add an `extended_definition` check to the grub2_enabled_fips OVAL for AES as there is a configuration change here. So, there needs to be:

1. An extended_definition check added to grub2_enable_fips_mode OVAL
2. A dracut-fips-aesni OVAL check in shared/checks that checks for AESNI enabled processors and that the `dracut-fips-aesni` package is installed.
3. The `dracut-fips-aesni` package needs to be added to the correct OS templates.